README.md

```diff
@@ -64,7 +64,18 @@ Navigate to the folder with subpostids.py and run the script. A CSV file will be
 
 #### subreddit.py ####
 
-NEW!!! Run `subreddit.py <subredditname>` and it will grab every submission and comment from that subreddit and store it in a sqlite database.  Furthermore, it will then parse all of the text in the comment bodies for an possible URL. if you rerun the script, it will avoid redownloading any comments or scripts so you can keep it updated.
+NEW!!! Run `subreddit.py <subredditname>` and it will grab every submission and comment from that subreddit and store it in a sqlite database. Furthermore, it will then parse all of the text in the comment bodies for a possible URL. if you rerun the script, it will avoid redownloading any comments or scripts so you can keep it updated.
+
+### Quickly get started with the redditPostArchiver Docker image
+
+You can run scripts by specifying the script name (with or without `.py`) and arguments. All output files will be at `/rpa` in the container. The credentials are passed through the environment variables `CLIENT_ID` and `CLIENT_SECRET`.
+
+#### Running Example
+
+`docker run -it --rm -e CLIENT_ID=YOUR_CLIENT_ID -e CLIENT_SECRET=YOUR_CLIENT_SECRET -v $(pwd)/rpa:/rpa yardenshoham/reddit-post-archiver postids GallowBoob`
```

Comment:
I was thinking we could use another account, perhaps a reddit-post-archiver account on Docker Hub and [set up automated builds](https://docs.docker.com/docker-hub/builds/).